afc-submit-wizard.js

```diff
@@ -747,7 +922,26 @@ function prepareTalkText(initialText) {
 
 	// remove |class=draft parameter in any WikiProject templates
 	text = text.replace(/(\{\{wikiproject.*?)\|\s*class\s*=\s*draft\s*/gi, '$1');
-
+	
+	sng = ui.sngInput.getValue().split(",");
+	sourcesExist = (sng[0] == "GNG") && (ui.bestsourcesInput1.getValue() || ui.bestsourcesInput2.getValue() || ui.bestsourcesInput3.getValue());
+	
+	if (sourcesExist) {
+		text = text.replace(/\{\{Best sources\|(.*?)\}\}/g, '');
```

Comment:
Does this really need to be on the talk page? I think we should just keep it on the draft page where all other submission-related comments go. Reviewers don't look at talk pages. Some character limits can be put in place to ensure the comments don't become too big.